packages/ethereum/ethereum-light-client/src/update.rs

```diff
@@ -15,9 +15,6 @@ pub fn update_consensus_state(
     current_client_state: ClientState,
     header: Header,
 ) -> Result<(u64, ConsensusState, Option<ClientState>), EthereumIBCError> {
-    let trusted_sync_committee = header.trusted_sync_committee;
-    let trusted_slot = trusted_sync_committee.trusted_slot;
-
     let consensus_update = header.consensus_update;
```

Comment:
Doesn't seem like a very useful variable, why not just use `header.consensus_update` everywhere
```suggestion
```

Reply:
addressed myself